climakitae/data_export.py

```diff
@@ -457,3 +457,109 @@ def _rchop(s, suffix):
+
+    # line 4 - ground temperatures, leave blank for now
+    line_4 = "GROUND TEMPERATURES\n"
+
+    # line 5 - holidays/daylight savings, leap year (yes/no), daylight savings start, daylight savings end, num of holidays
+    line_5 = "HOLIDAYS/DAYLIGHT SAVINGS,No,0,0,0\n"
+
+    # line 6 - comments 1, going to include simulation + scenario information here
+    line_6 = "COMMENTS 1,Typical meteorological year data produced on the Cal-Adapt: Analytics Engine, Scenario: {0}, Simulation: {1}\n".format(
+        df["scenario"].values[0], df["simulation"].values[0]
+    )
+
+    # line 7 - comments 2, putting the data variables here manually as they are not specified in epw format, and we are not including all
+    line_7 = "COMMENTS 2,Air Temperature at 2m,Dew point temperature,Relative humidity,Instantaneous downwelling shortwave flux at bottom,Shortwave surface downward diffuse irradiance,Instantaneous downwelling longwave flux at bottom,Wind speed at 10m,Wind direction at 10m,Surface Pressure\n"
+
+    # line 8 - data periods, num data periods, num records per hour, data period name, data period start day of week, data period start (Jan 1), data period end (Dec 31)
+    line_8 = "DATA PERIODS,1,1,Data,,1/1,12/31\n"
+
+    headers = [line_1, line_2, line_3, line_4, line_5, line_6, line_7, line_8]
+
+    return headers
+
+
+def write_tmy_file(filename_to_export, df, station_name, file_ext="tmy"):
+    """Exports TMY data either as .epw or .tmy file
+
+    Paramters
+    ---------
+    filename_to_export (str): Filename string, constructed with station name and simulation
+    df (pd.DataFrame): Dataframe of TMY data to export
+    station_name (str): Station name string
+    file_ext (str, optional): File extension for export, default is .tmy, options are "tmy" and "epw"
+
+    Returns
+    -------
+    None
+    """
+
+    # typical meteorological year format
+    if file_ext == "tmy":
+        path_to_file = filename_to_export + ".tmy"
+
+        with open(path_to_file, "w") as f:
+            f.writelines(tmy_header(station_name, df))  # writes required header lines
+            df = df.drop(
+                columns=["simulation", "lat", "lon", "scenario"]
+            )  # drops header columns from df
+            dfAsString = df.to_csv(sep=",", header=True, index=False)
+            f.write(dfAsString)  # writes file
+
+    # energy plus weather format
+    elif file_ext == "epw":
+        path_to_file = filename_to_export + ".epw"
+        with open(path_to_file, "w") as f:
+            f.writelines(epw_header(station_name, df))  # writes required header lines
+            df = df.drop(
+                columns=["simulation", "lat", "lon", "scenario"]
+            )  # drops header columns from df
+            dfAsString = df.to_csv(sep=",", header=False, index=False)
+            f.write(dfAsString)  # writes file
+
+    else:
+        print('Please pass either "tmy" or "epw" as a file format for export.')
```

Comment:
It might be nice if the function also printed something if the file was successfully exported, i.e 
" TMY file exported to .epw format with filename blahblablah.epw"